test/e2e/dynamic_provisioning_test.go

```diff
@@ -737,6 +737,104 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 		}
 		test.Run(cs, ns)
 	})
+
+	ginkgo.It("Should test pod failover with cordoning a node", func() {
+		skipIfUsingInTreeVolumePlugin()
```

Comment:
We should skip this test for multi-az since the scheduler won't be able to start the pod on the other node because it is in a different zone that the disk. This is why the `pull-azuredisk-csi-driver-e2e-multi-az` fails currently.

```console
May 25 08:47:50.985: INFO: At 2021-05-25 08:47:40 +0000 UTC - event for azuredisk-volume-tester-cgncb-86c7fcdbff-smnp4: {default-scheduler } FailedScheduling: 0/3 nodes are available: 1 node(s) had taint {node-role.kubernetes.io/master: true}, that the pod didn't tolerate, 1 node(s) had taint {node.kubernetes.io/unschedulable: }, that the pod didn't tolerate, 1 node(s) had volume node affinity conflict.
```